apps/user_ldap/lib/User/User.php

```diff
@@ -512,6 +574,51 @@ private function verifyQuotaValue(string $quotaValue) {
 		return $quotaValue === 'none' || $quotaValue === 'default' || \OC_Helper::computerFileSize($quotaValue) !== false;
 	}
 
+	/**
+	 * takes values from LDAP and stores it as Nextcloud user profile value
+	 * 
+	 * @param array $profileValues associaive array of property keys and values from LDAP
+	 * @param string|null $profileScope the scope of visibility to set
+	 */
+	private function updateProfile(array $profileValues, ?string $profileScope=null): void {
+		// check if user profile was refreshed before
+		if ($this->wasRefreshed('profile')) {
+			return;
+		}
+		// fetch/prepare user
+		$user = $this->userManager->get($this->uid);
+		if (is_null($user)) {
+			return;
+		}
+		// prepare AccountManager and Account
+		$accountManager = \OC::$server->get(IAccountManager::class);
```

Comment:
Is this a good way to get an accountManager instance, to work with?

Reply:
```suggestion
		$accountManager = Server::get(IAccountManager::class);
```
This with a `use OCP\Server` at the beginning of the file is the preferred way now.
The best is to use DI (Dependency Injection) in the constructor, but here for a user object it’s more complicated because it’s not a singleton. In this instance I think it’s fine to use a get here.